### PR TITLE
Reduce compulsory field for the addstu command

### PR DESCRIPTION
**Pull request by ForAeons**
# Change
- UI: Move nusnet up in the person card

# Breaking change
- addstu now only takes name and nusnet as compulsory argument

# Remarks
- Not sure how to do conditional rendering in javafx. For now, any optional fields are rendered with placeholders (this person does not have <field_name>)

Resolves: https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/44

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -100,8 +100,9 @@ Format: `addstu n/NAME p/PHONE_NUMBER e/EMAIL nn/NUSNET_ID a/ADDRESS [t/TAG]…
 </box>
 
 Examples:
-* `addstu n/John Doe p/98765432 e/johnd@example.com nn/e1234567 a/John street, block 123, #01-01`
-* `add n/Betsy nn/e01234567 Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `addstu n/John Doe nn/e1234567 [p/98765432] [e/johnd@example.com] [a/John street, block 123, #01-01]`
+* `add n/Betsy Crowe nn/e01234567 t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/Betsy Crowe nn/e01234567`
```
Comment (tituschewxj) [inline docs/UserGuide.md:105]: Perhaps you can add some description on what each command does, like the examples in `Editing a person`.

**Review by tituschewxj — APPROVED** [review]
LGTM! I only have a minor comment regarding the user guide example, other than that the code quality looks good for now.

File: docs/UserGuide.md (hunk 1 above)
Comment (ForAeons) [reply docs/UserGuide.md]: Thanks a lot for the review, I had added the examples.

**Review by belligerentbeagle — APPROVED** [review]
LGTM! Thanks so much for working on this feature! I think it has great impact on our usability for our target audience. 

A minor consideration I would like to suggest is - should the placeholder message be shorter? remove the "this person has"? What do you think?

**Comment by ForAeons** [comment]
> LGTM! Thanks so much for working on this feature! I think it has great impact on our usability for our target audience.
> 
> A minor consideration I would like to suggest is - should the placeholder message be shorter? remove the "this person has"? What do you think?

Updated, thanks for the review

**Pull request merged**